service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -22,6 +27,25 @@ public RunsApiController(CromwellServerConfiguration cromwellConfig) {
     this.cromwellConfig = cromwellConfig;
   }
 
+  @Override
+  public ResponseEntity<RunLogResponse> getRun() {
+    List<LogRunRequest> runs = new ArrayList<>();
+    String runId = UUID.randomUUID().toString();
+    String name = "CBAS";
+    Date now = new Date();
+
+    runs.add(
+        (new LogRunRequest()
```

Comment:
You can skip this paren pair (unless you are keeping consistent style with how other HTTP request handlers are written in this codebase, or something like that)